frontend/src/features/deploy/state/deployActions.ts

```diff
@@ -3,17 +3,41 @@ import { NewWorkflow, Workflow, workflowInfoFromWorkflow, workflowScriptString }
 import { WorkflowInfoAction } from '../../workflow/state/workflowActions'
 import { DEPLOY_START, DEPLOY_STOP } from './deployState'
 
+// const deployParams = {
+//   "apply": true,
+//   "workflow": {
+//     "deployed": true,
+//     "triggers": [
+//       {"type": "cron", "enabled": true, "periodicity": "R/PT1D" }
+//     ]
+//   },
+//   "ref": "me/example_one",
+//   "dataset": {
+//     "transform" : {
+//       "steps": [
+//         {
+//           "name": "transform",
+//           "syntax": "starlark",
+//           "category": "transform",
+//           "script": "def transform(ds, ctx):\n  ds.set_body([[1,2,3], [4,5,6], [7,8,9]])"
+//         }
+//       ]
+//     }
+//   }
+// }
+
 export function deployWorkflow(w: Workflow): ApiActionThunk {
   return async (dispatch, getState) => {
     return dispatch({
       type: 'deploy',
       
       [CALL_API]: {
-        endpoint: 'deploy',
+        endpoint: '/auto/deploy',
```

Comment:
possible different cloud endpoint

Reply:
Cloud doesn't have this one yet. But there is probably a long list of endpoints that generally do need to be brought over from core into cloud too. I'll make an issue for myself to review the entire API spec and filter out the ones that need to be added.